### ISSUE TITLE
Training data of pre-trained models

### ISSUE THREAD

**Issue by linakrisztian** [AWF-GAUG/TreeCrownDelineation#3]
Hello,

you have done a nice job here and very well documented.

However I still have a question concerning the documentation of the pre-trained models. 
Within the readme.md (downloaded with the pre-trained weights), the training data are listed as 89 images of 9 ha each, randomly sampled within Germany with a resolution of 20 cm.
In the paper (Individual tree crown delineation in high-resolution remote sensing images based on U-Net) two other datasets are listed. The Bengaluru satellite images (35 tiles of 9 ha each with 30 cm resolution) and Germany aerial images (39 tiles of 2500 m² each with 5 cm resolution). 

So my questions is, are the data used for the pre-trained models actually different to the one documented in the paper?

**Comment by maxfreu** [comment]
Hi! Sorry for the late reply, for some reason I didn't receive messages on activity. To answer your question: The pre-trained models use exactly the same training data as in the paper. As of now, only the 20cm aerial and the 30cm satellite model weights are uploaded.

**Comment by linakrisztian** [comment]
No problem, thanks for your reply. But it's still not completely clear to me: In the paper I could not find 20 cm aerial images mentioned. Have you resampled to 5cm?

**Comment by maxfreu** [comment]
Oh yes, you are right! There is another 20cm aerial dataset, which was not in the paper. I've trained the model on that and published those weights. The 5cm weights are not published. Do you need them?

**Comment by linakrisztian** [comment]
Ah ok. So the 20cm weights worked pretty well for us. It was just for clarification of data source. 
If it's not a big effort to publish the 5cm weights I guess it would be great to test and compare them. But depends on the effort.

**Comment by maxfreu** [comment]
I can deliver the weights earliest in May :(

**Comment by DecaiJin** [comment]
I observe the pretrained models contain 20cm weights(aerial, RGB-Nir) and 30cm weight(wv-3, 7band).  I want to ask some detailed info about these weights.  What were the used 7 channels of WV-3(8 bands) and what's the order of input bands. In addtion, what's the range of bands, e.g., 0-1 (reflectances), or 0-255( i saw the "--divided_by" 255), or 0-10000.

**Comment by maxfreu** [comment]
The WV3 coastal band was deleted. So the bands used are blue to IR + NDVI (red/red edge) in this order. The NDVI was normalized to 0..1, but the WV3 data was kept raw, unnormalized. I have written a small readme and zipped it along with the model weights. Furthermore I have published the training code, you now find it in the examples folder.
